CHANGELOG.md

```diff
@@ -5,8 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## Unreleased
+## [Unreleased]
```

Comment:
why the []s

Reply:
It's a [reference-style link](https://www.markdownguide.org/basic-syntax/#reference-style-links). And it seems that `[Unreleased]` is interpreted as `[Unreleased][Unreleased]` (cf. "[The first part of a reference-style link is formatted with two sets of brackets](https://www.markdownguide.org/basic-syntax/#reference-style-links:~:text=The%20first%20part%20of%20a%20reference%2Dstyle%20link%20is%20formatted%20with%20two%20sets%20of%20brackets)").